AutomatedTesting/Gem/PythonTests/editor/TestSuite_Main_Optimized.py

```diff
@@ -73,3 +73,6 @@ class test_Menus_ViewMenuOptions_Work(EditorSharedTest):
     @pytest.mark.skip(reason="Times out due to dialogs failing to dismiss: LYN-4208")
     class test_Menus_FileMenuOptions_Work(EditorSharedTest):
         from .EditorScripts import Menus_FileMenuOptions as test_module
+    
+    class test_Sample_Editor_Tests(EditorSharedTest):
+        from .EditorScripts import Sample_Editor_Tests as test_module
```

Comment:
Needs a newline at the end of the file here.

Reply:
fixed